integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseUtils.java

```diff
@@ -277,8 +278,10 @@ public static int call(Logger logger, FuseCallable callable, String methodName,
     long startMs = System.currentTimeMillis();
     int ret = callable.call();
     long durationMs = System.currentTimeMillis() - startMs;
-    MetricsSystem.timer(methodName).update(durationMs, TimeUnit.MILLISECONDS);
     logger.debug("Exit ({}): {}({}) in {} ms", ret, methodName, debugDesc, durationMs);
+    MetricsSystem.timer(methodName).update(durationMs, TimeUnit.MILLISECONDS);
```

Comment:
Would you please reference `MetricsSystem.MultiTimerContext` invented by #13671

Reply:
Added here because we already recorded the time duration for logging, no need to use more timerContext for repeated time recording